Free_Games.py

```diff
@@ -66,16 +67,16 @@ def getGame():
         return
     
     try:
-        if not wait_until_xpath_clickable_then_click("//*[text() = 'Get']"):
-            raise TypeError("Unable to find 'Get' button")
+        if not wait_until_xpath_clickable_then_click("//*[text() = 'Get']", 30):
+            print("Unable to find 'Get' button")
     except:
         print("Failed to click on 'Get' button. Using parent element workaround... ", end='')
-        if not wait_until_xpath_clickable_then_click("//*[text() = 'Get']/.."):
-            raise TypeError("Workaround failed")
+        if not wait_until_xpath_clickable_then_click("//*[text() = 'Get']/..", 30):
+            print("Workaround failed")

```

Comment:
What I meant is that we could be using some sort of infinite loop, just like we are doing currently with waiting for captcha completion.  
Check for the element with a 1 second duration. If after xx loop iterations (10?) we still didn't find the element, print in the console that we are still waiting to find the element. If after xx loop iterations (60+?) still nothing, then raise an error to exit.

Reply:
Could you explain a little how the looping solution is better than the delay thing you have?

Reply:
It wouldn't, simply (lol). Now that I think about it, it'd be quite stupid actually.  
Though we definitely need to raise an error if the element was not found after those 30 seconds (maybe higher? in case of slower website or connection - it doesn't affect the script in normal circumstances so it'd be fine anyway) so that the script stops.

Reply:
Nice lol And just to be clear, are you saying we should leave that 30 second delay? If so, should it be the parent one inside the except? Or both of them just for fun?

Reply:
This won't fix anything in my issue. This adds delay to "get" button while I am having button with "order" button.

Reply:
It does though...
Click on the [commit](https://github.com/MasonStooksbury/Free-Games/pull/34/commits/9602e490dbfbe9f6d4c83857709781450d4cfe6c), line 78

Reply:
You're correct, but these methods are all close to each other, and if you are having issues with "order", then it's possible you or others might have issues with the "get" at some point. We're discussing here how we might better fix the whole lot of them since they are next to each other.

Reply:
After looking through the code I'm planning on having fun making the code asynchronous. To where not concrete delays are written WebDriverWait + async await functions should clean the code up a lot. plus i plan on finding a solution to safely storing the users password and 2fa